rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -931,11 +928,21 @@ error_code sys_usbd_get_isochronous_transfer_status(ppu_thread& ppu, u32 handle,
 	return CELL_OK;
 }
 
-error_code sys_usbd_get_device_location(ppu_thread& ppu)
+error_code sys_usbd_get_device_location(ppu_thread& ppu, u32 handle, u32 device_handle, vm::ptr<u8> location)
 {
 	ppu.state += cpu_flag::wait;
 
-	sys_usbd.todo("sys_usbd_get_device_location()");
+	sys_usbd.notice("sys_usbd_get_device_location(handle=0x%x, device_handle=0x%x, location=*0x%x)", handle, device_handle, location);
```

Comment:
Missing param check for location

Reply:
It's a syscall, there is generally no parameter check.
There is one in cellUsbd function which is lled.